javascript/ql/src/semmle/javascript/dataflow/TaintTracking.qll

```diff
@@ -612,6 +612,26 @@ module TaintTracking {
 
   }
 
+  /** A check of the form `if(~o.indexOf(x))`, which sanitizes `x` in its "then" branch. */
```

Comment:
It may be helpful to add an example here to explain why it's `x` that is sanitised and not `o`. (In fact, it seems like there are uses of this idiom that sanitise `o`, but they are perhaps more difficult to capture.)

Reply:
Perhaps all that's needed is to rename `o` to `whitelist`. But please also explain that this is equivalent to `if(whitelist.indexOf(x) !== -1)`.

Reply:
Done. The `o` in the `o.indexOf(x) !== -1` case has also been renamed to `whitelist`.